app/pdf-viewer/buffer.py

```diff
@@ -162,7 +162,7 @@ def copy_select(self):
             self.message_to_emacs.emit(content)
             self.buffer_widget.cleanup_select()
         else:
-            self.message_to_emacs.emit("Cannot copy, you haven't select anything!")
+            self.message_to_emacs.emit("Cannot copy, you should double click left mouse button then drag to select pdf text!")
```

Comment:
you should double click your mouse and hover through the text on the PDF. Don't click and drag!

Reply:
s/Don't/Do not